store/mockstore/mocktikv/rpc.go

```diff
@@ -723,9 +733,21 @@ func (c *RPCClient) checkArgs(ctx context.Context, addr string) (*rpcHandler, er
 	return handler, nil
 }
 
-// TiDBRPCServerCoprocessorHandler is the TiDB rpc server coprocessor handler.
+// GRPCClientFactor is the GRPC client factory.
+// Use global variable to avoid circle import.
 // TODO: remove this global variable.
-var TiDBRPCServerCoprocessorHandler func(context.Context, *coprocessor.Request) *coprocessor.Response
+var GRPCClientFactor func() Client
```

Comment:
Why naming it 'Factor'?

Reply:
Sorry, It's a mistake, It should be `factory`.